From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Tue, 3 Sep 2019 13:09:31 -0500
Subject: [PATCH] liblocaledir is located in /usr/share/local

glibc locale location was moved into /usr/share [1]
so make this package to reference the locales in such location
instead.

[1]-https://github.com/clearlinux-pkgs/glibc/commit/bb775f3f1fbc03816138f1278c5cba797030f145

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 1dc8a73..84a3dc2 100644
--- a/meson.build
+++ b/meson.build
@@ -41,7 +41,7 @@ prefix = get_option('prefix')
 datadir = prefix / get_option('datadir')
 libexecdir = prefix / get_option('libexecdir')
 
-liblocaledir = get_option('prefix') / 'lib/locale'
+liblocaledir = datadir / 'locale'
 localedir = datadir / 'locale'
 test_metadir = datadir / 'installed-tests' / meson.project_name()
 test_execdir = libexecdir / 'installed-tests' / meson.project_name()
